### DISCUSSION TITLE
Vercel tracked only first commit

### DISCUSSION THREAD

**Discussion by lorenzoceglia** [vercel/community#2097] [Help] [answered]
Hi guys I linked a private repo to an instance in Vercel, so it took the first commit on master and created the instance. 
I pushed other commits but the redeploy doesnt trigger. If I check on Vercel I see that the last commit is the first, on github I have the others commits and if I hover the mouse on them it says : 
All checks have passed
1 successful check
@vercel
Vercel — Deployment has completed

Could you help me? Thanks in advance!
<img width="850" alt="Screenshot 2023-04-16 alle 21 39 09" src="https://user-images.githubusercontent.com/34025272/232337901-d1b13172-9866-45a1-8610-f21f3588ffd7.png">
<img width="857" alt="Screenshot 2023-04-16 alle 21 39 27" src="https://user-images.githubusercontent.com/34025272/232337919-da94e6b1-234a-4ab5-9f2f-3f78b560c48e.png">

**Comment by amyegan** [comment]
Hi @lorenzoceglia! When you push new commits to GitHub, a new deployment should be triggered rather than redeployment of an existing build. Do you not have a new build in the Deployments tab of your Project Dashboard?

**Reply by lorenzoceglia** [reply, marked as answer]
Yeah It was stuck on the initial commit. Anyway I fixed the issue simply renaming the master branch to main. Thanks you so much for the reply!
